dts/bindings/display/solomon,ssd1322.yaml

```diff
@@ -0,0 +1,63 @@
+description: SSD1322 display controller
+
+compatible: "solomon,ssd1322"
+
+include: [mipi-dbi-spi-device.yaml, display-controller.yaml]
+
+properties:
+  column-offset:
+    type: int
+    required: true
+    description: First visible column number.
+
+  row-offset:
+    type: int
+    default: 0
```

Comment:
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values